qemu/tests/cgroup.py

```diff
@@ -2059,6 +2061,51 @@ def memory_move():
         else:
             return (out)
 
+    def cfs_bandwidth():
+        """
+        CFS bandwidth limit
+        """
+        def stress_thread():
+            session.cmd("stress -c 1 -t 100", 120)
+
+        logging.info("Setup test")
+        modules = CgroupModules()
+        if (modules.init(['cpu']) != 1):
+            raise exceptions.TestFail("Can't mount cpu cgroup modules")
+        cgroup = Cgroup('cpu', '')
+        cgroup.initialize(modules)
+
+        # Create VM with smp=1
+        params['smp'] = 1
+        params["vcpu_sockets"] = 1
+        vm_name = params["main_vm"]
+        params["start_vm"] = "yes"
+        env_process.preprocess_vm(test, params, env, vm_name)
+        vm = env.get_vm(params["main_vm"])
+        login_timeout = int(params.get("login_timeout", 360))
+        session = vm.wait_for_login(timeout=login_timeout)
+
+        cgroup.mk_cgroup()
+        cgroup.set_property("cpu.cfs_period_us", 100000, 0)
+        cgroup.set_property("cpu.cfs_quota_us", 50000, 0)
+        assign_vm_into_cgroup(vm, cgroup, 0)
+
+        logging.info("Test")
```

Comment:
Please elaborate more than just a "Test" .
And line `logging.info("Setup test")`,  what did you setup for the test?